### PR TITLE
Fix threading for argon2.

### PR DESCRIPTION
**Pull request by benjamin-tlon**

### CODE REVIEW

**Comment by benjamin-tlon** [comment]
https://github.com/P-H-C/phc-winner-argon2/issues/257

File: jets/e/argon2.c [hunk 1]
```diff
@@ -109,7 +146,12 @@
     }
 
     u3z(key); u3z(extra); u3z(dat); u3z(sat);
-    return u3kc_rev(3, out, u3i_bytes(out, outhash));
+
+    // Copy result from the stack.
+    u3_noun result = u3i_bytes(out, output_buf);
+
+    // XX Do we need to free `result`?
+    return u3kc_rev(3, out, result);
```
Comment (joemfb) [inline jets/e/argon2.c]: It's implicitly `u3z`'d, because `u3k*` jets have transfer semantics.

Comment (benjamin-tlon) [reply jets/e/argon2.c]: Ah! Ok, that makes sense. Thanks.

**Comment by joemfb** [comment]
@benjamin-tlon, was this intended to go in, or where these changes just for experimenting with CI?

**Comment by joemfb** [comment]
Can we close this?

**Pull request closed without merging**